lintconf.yaml

```diff
@@ -0,0 +1,42 @@
+---
+rules:
+  braces:
+    min-spaces-inside: 0
+    max-spaces-inside: 0
+    min-spaces-inside-empty: -1
+    max-spaces-inside-empty: -1
+  brackets:
+    min-spaces-inside: 0
+    max-spaces-inside: 0
+    min-spaces-inside-empty: -1
+    max-spaces-inside-empty: -1
+  colons:
+    max-spaces-before: 0
+    max-spaces-after: 1
+  commas:
+    max-spaces-before: 0
+    min-spaces-after: 1
+    max-spaces-after: 1
+  comments:
+    require-starting-space: true
+    min-spaces-from-content: 2
+  document-end: disable
+  document-start: disable           # No --- to start a file
```

Comment:
Why are you disabling this check? Unless there is a reason I would prefer to enforce YAML document start requirement.

Reply:
I also think it conflicts with a different linter that is requiring it!

Reply:
I followed the instructions from https://redhat-cop.github.io/ci/linting-testing-helm-charts.html

I can try to disable it.

Note that the `lintconf.yaml` is used by ct to just check:
- values.yaml
- Chart.yaml

The rest is still being checked by the other yamllint

Reply:
I use the `.yamllint` file for configuration to prevent having duplicated and different files.